icons/gpu.json

```
@@ -0,0 +1,20 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "xandykati98",
+    "karsa-mistmere"
+  ],
+  "tags": [
+    "processor",
+    "cores",
+    "technology",
+    "computer",
+    "chip",
+    "circuit",
+    "specs",
+    "gpu"
```

`gpu` as a tag is redundant, we could add some other tags (synonyms, possible use cases) though:
```suggestion
    "graphics processing unit",
    "video card",
    "display adapter",
    "gddr",
    "rendering",
    "digital image processing,"
    "crypto mining"
```